raiden/ui/cli.py

```diff
@@ -187,6 +245,13 @@ def convert(self, value, param, ctx):
         default=True,
     ),
     click.option(
+        '--sync-check/--no-sync-check',
+        help=(
+            'Enable/Disable usage of an oracle to check if the ethereum node is synchronized'
```

Comment:
maybe specify the oracle's source here (etherscan?). Would be helpful for the users to know.

Reply:
If there is no other way to change the default behaviour you can omit the `/--no-option` and use a simple one with `is_flag=True`. More readable IMO.

Reply:
@ulope I don't think I understood what you want.

The default behavior is to do the check. The `--no-sync-check` was added mostly to skip the test on the build, we don't have a running etheruem node on the before_script and we definitely don't want to sync, it may also be useful to someone on a private chain.

Reply:
Note: I just realized that smoketest is not decorated with `options`, so the swich `--no-sync-check` would only be useful for private chains.